.travis.yml

```diff
@@ -7,9 +7,16 @@ python:
   - "pypy"
 
 install:
-  - "python setup.py --version"
-  - "travis_retry pip install -r requirements.txt"
-  - echo ${TRAVIS_PYTHON_VERSION}
+  - travis_retry pip install -r requirements.txt
+  - travis_retry pip install python-coveralls coverage nose-cov
+  - echo TRAVIS_PYTHON_VERSION=${TRAVIS_PYTHON_VERSION}
   - if [ ${TRAVIS_PYTHON_VERSION:0:1} != "3" ]; then travis_retry pip install contextlib2; fi
+  - python setup.py --version
 
-script: nosetests
+# Run test
+script:
+  - nosetests -v --with-cov --cov streamparse --cov-config .coveragerc --logging-level=DEBUG
+
+# Calculate coverage on success
+after_success:
+  - coveralls --config_file .coveragerc
```

Comment:
I've been deleting them, but we're actually getting duplicate coveralls comments because it's commenting for every single Travis job that succeeds (i.e., 2.7, 3.3, 3.4, and pypy), so we should probably just pick a version and have coveralls only get called for that version of Python.